package.json

```diff
@@ -46,6 +47,7 @@
     "normalize.css": "^8.0.1",
     "postcss": "^7.0.31",
     "preact": "10.4.5",
+    "prop-types": "^15.7.2",
```

Comment:
This dependency was previously omitted by mistake. Importing it worked because another dependency happened to rely on it.